manylinux/generate_wheels.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 cd /tmp/katsdpimager
 mkdir -p /output
-for d in /opt/python/cp{36,37,38}*; do
+for d in /opt/python/cp{36,37,38,39}*; do
```

Comment:
You are a brave man... I tried 3.9 once and ran away screaming when I think numba + llvmlite got involved :-D